package-lock.json

```diff
@@ -0,0 +1,4809 @@
+{
```

Comment:
Should this file be `.gitignore`, or do we prefer to have it here for reference?

Reply:
@edthedev the `package-lock.json` is required to ensure the unit tests are repeatable. It is also a Github recommended best practice for JavaScript actions to include the `package-lock.json` file in version control:

> From your terminal, commit your `action.yml`, `index.js`, `node_modules`, `package.json`, `package-lock.json`, and `README.md` files. If you added a `.gitignore` file that lists `node_modules`, you'll need to remove that line to commit the `node_modules` directory.
https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action

Note that I did not check in the `node_modules` directory because it is not required during runtime. For this project `node_modules` directory is needed only during testing. The `package-lock.json` is [adequate](https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json) for rebuilding the `node_modules` when needed for testing:

> This file is intended to be committed into source repositories, and serves various purposes:
> 
> * Describe a single representation of a dependency tree such that teammates, deployments, and continuous integration are guaranteed to install exactly the same dependencies.
> * Provide a facility for users to "time-travel" to previous states of node_modules without having to commit the directory itself.
https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json